docs/pbc/all/tax-management/202400.0/base-shop/vertex/install-vertex.md

```diff
@@ -546,9 +554,71 @@ class ProductOfferStockDependencyProvider extends SprykerProductOfferStockDepend
     protected function getStockTransferExpanderPluginCollection(): array
     {
         return [
+            # This plugin is providing warehouse address to StockTransfer objects which is used during tax calculation.
             new StockAddressStockTransferProductOfferStockExpanderPlugin(),
         ];
     }
 }
 
 ```
+
+### 5. Configure Shop Application dependency provider
+
+Add the following to `src/Pyz/Yves/ShopApplication/ShopApplicationDependencyProvider.php`:
+
+```php
+
+namespace Pyz\Yves\ShopApplication;
+
+use SprykerShop\Yves\ShopApplication\ShopApplicationDependencyProvider as SprykerShopApplicationDependencyProvider;
+use SprykerShop\Yves\CartPage\Widget\CartSummaryHideTaxAmountWidget;
+
+class ShopApplicationDependencyProvider extends SprykerShopApplicationDependencyProvider
+{
+    /**
+     * @phpstan-return array<class-string<\Spryker\Yves\Kernel\Widget\AbstractWidget>>
+     *
+     * @return array<string>
+     */
+    protected function getGlobalWidgets(): array
+    {
+        return [
+            // ...
+
+            # This widget is replacing Spryker OOTB tax display in cart summary page with text stating that tax amount will be calculated during checkout process.
+            CartSummaryHideTaxAmountWidget::class,
+        ];
+    }
+}
+
+```
+
+
+### 6. Configure Message Broker dependency provider
+
+Add the following to `src/Pyz/Zed/MessageBroker/MessageBrokerDependencyProvider.php`:
```

Comment:
these paragraphs have to go under `1. Integrate ACP connector module for tax calculation`, where all code/configs for SCOS parts are provided.